src/applications/vaos/new-appointment/components/DateTimeRequestPage/index.jsx

```diff
@@ -143,10 +142,11 @@ export function DateTimeRequestPage({
         id="optionTime"
         renderIndicator={props => <SelectedIndicator {...props} />}
         renderOptions={props => <DateTimeRequestOptions {...props} />}
-        validationError={
-          submitted || isMaxSelectionsError(validationError)
-            ? validationError
-            : null
+        required
+        requiredMessage={validationError}
```

Comment:
I think when I wrote this ticket I missed that we had another validation case on the request page. Or I was assuming that we would want to remove the selected too many dates message in favor of not allowing you to choose more than the max.

In any case, my intent with these changes was to try to push most of the logic about determining when there's a validation error into the Calendar component itself, and have the page components past custom messages (if we need to) and manage the submitted state. 

So instead of passing `validationError` here, what if we made another message prop like `maxSelectionsMessage` and moved the `exceededMaxSelections(data.selectedDates)` logic into the Calendar components?

Reply:
Yeah, that looks good to me